src/games/game.h

```diff
@@ -847,6 +847,8 @@ class GameRep : public BaseGameRep {
   //@{
   /// Set the probability distribution of actions at a chance node
   virtual Game SetChanceProbs(const GameInfoset &, const Array<Number> &) = 0;
+  /// Normalize the probability distribution of actions at a chance node
```

Comment:
Having a separate `NormaliseChanceProbs` function makes sense, but it doesn't need to be a member of `GameRep`.  It just needs to be present in `TreeGameRep` as that's the only representation (at least at the moment) that implements the operation.   That means changes to the other representation classes can be rolled back.

Further, within `TreeGameRep`, this should be a private member function; there's no reason for external code to call this as having the probabilities be summing to one should already be maintained as an invariant.